examples/social-media/readme.md

```diff
@@ -0,0 +1,24 @@
+## Small social media example
+
+Say we have a social network, a very basic one, where all you can do is upvoting comments.
+
+The individual services that need to be defined are:
+
+- A frontend application where users interact through the website, where they can like comments.
+- A backend WebSocket server that sends and receives events for the UI to update in real-time.
+- A comment service which processes all events related to comments through a message broker.
+- A public-facing API which allows others to get notified about updates.
```

Comment:
It is not clear enought for me what this public facing API would do. Here you mention `allows others to get notified about updates`, however in the `public.asyncapi.yaml` file, there is only one operation which is `publish`. Shall we add a `Subscribe` Operation so users can be notified?

Reply:
The public API needs to be seen as an external application, that anyone can have and interact with the system. We still use `publish` because that application need to consume messages, where others publish to it 🙂